backend/capellacollab/settings/modelsources/t4c/repositories/interface.py

```diff
@@ -61,3 +64,56 @@ def stop_repository(instance: DatabaseT4CInstance, name: str):
     )
     r.raise_for_status()
     return r.json()
+
+
+def get_t4c_status(instance: DatabaseT4CInstance):
```

Comment:
This function is independent from T4C repositories. It should be on the T4C instance level.